macros/src/main/scala/mainecoon/autoFlatMap.scala

```diff
@@ -134,6 +115,27 @@ object autoFlatMap {
     cls.copy(companion = cls.companion.addStats(instanceDef))
 
   }
+
+  private[mainecoon] def mapMethods(template: Template): List[Defn.Def] = {
```

Comment:
build failed. we need to pass in `effectTypeName` as an argument

Reply:
You're right, don't know how this slipped through.